pycsob/client.py

```diff
@@ -17,6 +20,202 @@ def send(self, request, **kwargs):
         return super(HTTPAdapter, self).send(request, **kwargs)
 
 
+class ConvertMixin:
+    """Convert instance into ordered dict."""
+    
+    def _to_camel_case(self, name: str) -> str:
```

Comment:
Wouldn't it be better to use the name in camel case format instead of this function?

Reply:
Well, so far all API methods (like `payment_init`, `get_payment_process_url`, etc.) use snake case for parameters in spite of the documentation. I wanted to follow the style that is why I implemented the conversion function.

If you insist on came case parameters we should change them throughout all API to be consistent, shouldn't we?

Reply:
I don't think it's necessary to change all the parameters on the camell case. We just need a case of nested items in `dataclass`.  Of course it would be better to have the names compact, but it saves a lot of code. Is it worth it? Don't you think?